React-Native/en/3.3.1.md

```diff
@@ -0,0 +1,23 @@
+# [Success Criterion 3.3.1 - Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html)
+
+In React Native there is no standard for indicating errors. You can use the [Text](https://reactnative.dev/docs/text) component to display an error. Whenever an error occurs, users of assistive technologies should also be able to identify this.
```

Comment:
Are there any packages for showing errors on input fields?

Reply:
I can't really find anything for it. I found a text input component from the react native paper package: https://callstack.github.io/react-native-paper/text-input.html. But you still need to use the helpertext component for that to display an error: https://callstack.github.io/react-native-paper/helper-text.html. So I doesn't really add to what we already describe here in my opinion.

Reply:
I think it's good to add as example

I think it's still useful information to add. The HelperText component seems to have a couple more features compared to a plain Text component.

Example:

You could use the [`HelperText`](https://callstack.github.io/react-native-paper/helper-text.html) component to add helper text. 

```jsx
<View>
  <TextInput label="Email" value={text} onChangeText={onChangeText} />
  <HelperText type="error" visible={hasErrors()}>
    Email address is invalid!
  </HelperText>
</View>
```